flink-runtime/src/test/java/org/apache/flink/runtime/execution/librarycache/FlinkUserCodeClassLoadersTest.java

```diff
@@ -208,6 +212,64 @@ public void testRepeatedParentFirstPatternClass() throws Exception {
         childClassLoader.close();
     }
 
+    @Test
+    public void testGetClassLoaderInfo() {
+        final String className = FlinkUserCodeClassLoadersTest.class.getName();
+        final String parentFirstPattern = className.substring(0, className.lastIndexOf('.'));
+
+        final ClassLoader parentClassLoader = getClass().getClassLoader();
+
+        final URL childCodePath = getClass().getProtectionDomain().getCodeSource().getLocation();
+
+        final URLClassLoader childClassLoader =
+                FlinkUserCodeClassLoaders.childFirst(
+                        new URL[] {childCodePath},
+                        parentClassLoader,
+                        new String[] {parentFirstPattern},
+                        NOOP_EXCEPTION_HANDLER,
+                        true);
+
+        String formattedURL = null;
+        try {
+            formattedURL = ClassLoaderUtil.formatURL(childCodePath);
+        } catch (IOException e) {
+            // Should not fail here
+            fail();
+        }
```

Comment:
```suggestion
        String formattedURL = ClassLoaderUtil.formatURL(childCodePath);
```
Let the exception bubble up

Reply:
It was fixed in  c927703